js/visearch.js

```diff
@@ -234,9 +225,9 @@
         const stop = new Date().getTime();
         console.log(`ViSearch ${path} finished in ${stop - start}ms`);
         if (options.track_enable && json.status === 'OK') {
-          sendEvent(EVENTS.actions, {
-            reqid,
-            action: path,
+          sendEvent("search", {
```

Comment:
this one is optional , no longer required. Since we will only check in API log.

Reply:
i dont really understand what you meant.
This is needed, this function will send the search events to the analytics tracker API.

And to send a search event, we need to wait for search to complete to get the `queryId`

Reply:
as mentioned just now, there is no need to send this event to Analytics log because it is already captured by ViSearch API log. This would be a duplicate "search" event.

Reply:
ok, understood.

i have remove the auto sending of tracking event on search